### PR TITLE
impr(quotes): Adding Swift Quotes File.

### PR DESCRIPTION
**Pull request by CedricBaaklini**
I've noticed that there is no quotes for Swift and as someone who loves Swift and uses it as their primary language, I'm a little disappointed that there is no quote file for Swift. So, I decided to make one myself and added a few quotes to get started, including a "Hello World" Program. Feel free to let me know if there are any errors or improvements to be made.

### CODE REVIEW

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/8101848500

File: frontend/static/quotes/test.ts
Comment (Miodec) [inline frontend/static/quotes/test.ts:1]: Remove this please

File: frontend/static/quotes/code_swift.json
Comment (Miodec) [inline frontend/static/quotes/code_swift.json:1]: Could you please add more quotes? It would greatly reduce repetition

**Review by Miodec — CHANGES_REQUESTED** [review]

File: frontend/static/quotes/code_swift.json
Comment (CedricBaaklini) [reply frontend/static/quotes/code_swift.json:1]: Sure thing. Like I said in the message, it is just a few to get started.

Comment (CedricBaaklini) [reply frontend/static/quotes/code_swift.json:1]: It would be nice for more people to help out add quotes into the file.

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/8103907201

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/8103952927

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/8115827165

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/8177153147

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/8177470246

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/8208980663

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/8224246235

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/8425326176

**Pull request closed without merging**